orchestration/_tests/test_globus_flow.py

```diff
@@ -0,0 +1,333 @@
+from typing import List, Optional, Dict, Any
+from uuid import UUID, uuid4
+import warnings
+
+from globus_compute_sdk.sdk.client import Client
+
+from prefect.testing.utilities import prefect_test_harness
+from prefect.blocks.system import JSON, Secret
+from pydantic import BaseModel, PydanticDeprecatedSince20
+import pytest
+from pytest_mock import MockFixture
+
+from .test_globus import MockTransferClient
+
+warnings.filterwarnings("ignore", category=DeprecationWarning)
+warnings.filterwarnings("ignore", category=PydanticDeprecatedSince20)
+
+
+@pytest.fixture(autouse=True, scope="session")
+def prefect_test_fixture():
+    """
+    A pytest fixture that automatically sets up and tears down the Prefect test harness
+    for the entire test session. It creates and saves test secrets and configurations
+    required for Globus integration.
+
+    Yields:
+        None
+    """
+    with prefect_test_harness():
+        globus_client_id = Secret(value=str(uuid4()))
+        globus_client_id.save(name="globus-client-id")
+
+        globus_client_secret = Secret(value=str(uuid4()))
+        globus_client_secret.save(name="globus-client-secret")
+
+        globus_compute_endpoint = Secret(value=str(uuid4()))
+        globus_compute_endpoint.save(name="globus-compute-endpoint")
+
+        globus_reconstruction_function = Secret(value=str(uuid4()))
+        globus_reconstruction_function.save(name="globus-reconstruction-function")
+
+        globus_iribeta_cgs_endpoint = Secret(value=str(uuid4()))
+        globus_iribeta_cgs_endpoint.save(name="globus-iribeta-cgs-endpoint")
+
+        globus_flow_id = Secret(value=str(uuid4()))
+        globus_flow_id.save(name="globus-flow-id")
+
+        pruning_config = JSON(value={"max_wait_seconds": 600})
+        pruning_config.save(name="pruning-config")
+        yield
+
+
+class FlowDefinition(BaseModel):
+    """Model for flow definition"""
+    StartAt: str
+    States: Dict[str, Dict[str, Any]]
+
+
+class FlowInputSchema(BaseModel):
```

Comment:
I would like to remove FlowInputSchema, FlowDefinition and the prefect imports.

Reply:
The previous comment had a typo. I meant to ask to remove the pydantic imports, which won't be needed when we remove the `BaseModel` sub-classes.